workflow/executor/executor.go

```diff
@@ -204,6 +204,11 @@ func (we *WorkflowExecutor) LoadArtifacts(ctx context.Context) error {
 		// the file is a tarball or not. If it is, it is first extracted then renamed to
 		// the desired location. If not, it is simply renamed to the location.
 		tempArtPath := artPath + ".tmp"
+		// Ensure parent directory exist, create if missing
+		tempArtDir := filepath.Dir(tempArtPath)
+		if err := os.MkdirAll(tempArtDir, 0o700); err != nil {
```

Comment:
question: why do we set this 700?

Reply:
The code here creates a temporary path, then artifact driver will load(download) the object to this path.
The minimum dir permission across the artifact drivers is 700.
Are you experiencing an issue with this?